src/commands/cli/tarballs/smoke.ts

```diff
@@ -54,14 +58,72 @@ export default class SmokeTest extends SfCommand<void> {
       this.execute(executable, '--version'),
       this.execute(executable, '--help'),
       this.execute(executable, 'plugins --core'),
-      this.execute(executable, 'plugins:install @salesforce/plugin-alias@latest'),
+      this.testInstall(executable, '@salesforce/plugin-alias', 'latest'),
     ]);
+    await this.testJITInstall(executable);
     await this.initializeAllCommands(executable);
   }
 
+  private async testJITInstall(executable: string): Promise<void> {
+    const fileData = await fs.promises.readFile('package.json', 'utf8');
+    const packageJson = parseJson(fileData) as PackageJson;
+    const jitPlugins = Object.keys(packageJson.oclif.jitPlugins ?? {});
+    if (jitPlugins.length === 0) return;
+
+    const manifestData = await fs.promises.readFile(path.join('tmp', this.flags.cli, 'oclif.manifest.json'), 'utf8');
+    const manifest = parseJson(manifestData) as unknown as Interfaces.Manifest;
```

Comment:
this is probably our #1 reason for all the TS assertions.  Maybe https://github.com/forcedotcom/kit/blob/5e1e5bf6befef811a1900b8e288710ac7a791142/src/json.ts#L28 should take a generic <T>  for simplicity?